migrator/MigratorService.java

```diff
@@ -32,17 +32,17 @@
 public class MigratorService extends MigratorGrpc.MigratorImplBase {
     private static final Logger LOG = LoggerFactory.getLogger(MigratorService.class);
 
-    private final TypeDB typedb;
+    private final TypeDB.DatabaseManager databaseMgr;
```

Comment:
Now that `RocksTypeDB` has been deleted, the reference is updated to point to `databaseManager`.